
# msg to set PID current point

uint8 DoF

float32 Current
